mails/themes/modern/core/productoutofstock.html.twig

```diff
@@ -189,7 +189,7 @@
                                     <tr>
                                       <td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;">
                                         <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:left;color:#363A41;">
-                                          {{ 'Replenish your inventory, go to the <span class="label">"Catalog" &gt; "Stocks"</span> section of your back office to manage your stock.'|trans({}, 'Emails.Body', locale)|raw }}
+                                          {{ 'Replenish your inventory, go to the <span class="label">"Catalog" > "Stocks"</span> section of your back office to manage your stock.'|trans({}, 'Emails.Body', locale)|raw }}
```

Comment:
```suggestion
                                          {{ 'Replenish your inventory, go to the <span class="label">Catalog > Stocks</span> section of your back office to manage your stock.'|trans({}, 'Emails.Body', locale)|raw }}
```

Reply:
Shouldn't it be the same balises (I don't know the English word, aha) than before?

Reply:
It is the same *tag* (😉) `<span class="label">` If you compare it with the classic theme where we use `<span><strong>` then yes it is normal, the theme css and integration are different. I'm not a big fan of including html in the translation though because it means they are not compatible with other themes..
But this would require to split the message into three parts, or maybe include placeholders like this:

`'Replenish your inventory, go to the [bold]Catalog > Stocks[\bold] section of your back office to manage your stock.'`